File: Application/Sources/UI/Helpers/UIVisualEffectView+PlaySRG.m
//
//  Copyright (c) SRG SSR. All rights reserved.
//
//  License information is available from the LICENSE file.
//

#import "UIVisualEffectView+PlaySRG.h"

@implementation UIVisualEffectView (PlaySRG)

#pragma mark Class methods

+ (UIVisualEffectView *)play_blurView
{
    UIBlurEffectStyle blurEffectStyle;
    blurEffectStyle = UIBlurEffectStyleSystemMaterialDark;
    
    UIVisualEffect *blurEffect = [UIBlurEffect effectWithStyle:blurEffectStyle];
    UIVisualEffectView *blurView = [[UIVisualEffectView alloc] initWithEffect:blurEffect];
    blurView.backgroundColor = nil;
    return blurView;
}

@end

File: Application/Sources/UI/Helpers/UIVisualEffectView+PlaySRG.swift
//
//  Copyright (c) SRG SSR. All rights reserved.
//
//  License information is available from the LICENSE file.
//

import UIKit

extension UIVisualEffectView {
    /**
     *  Standard view with blur effect consistent with Play look and feel.
     */
    @objc static var play_blurView: UIVisualEffectView {
        let blurView = UIVisualEffectView(effect: UIBlurEffect(style: .systemMaterialDark))
        blurView.backgroundColor = nil
        return blurView
    }
}
